<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>545</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>545</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OPCUA Server Python</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/server.png</normaloff>icons/server.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
 QMainWindow {
        padding: 5px;
    }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>504</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>504</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(234, 234, 234);</string>
      </property>
      <property name="text">
       <string>Servidor OPC UA Python</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="StatusServer">
      <property name="minimumSize">
       <size>
        <width>504</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>504</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 0);</string>
      </property>
      <property name="text">
       <string> RESULT:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>504</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>504</width>
        <height>580</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane {
        border: 4px solid rgb(171, 171, 171);
		
        top: -1px;
    }
QTabBar::tab:selected {
        
		background-color: rgb(171, 171, 171);
	    color: black;
      
    }
 QTabBar::tab:hover {
           /* azul claro no hover */
		background: rgb(180, 191, 209);
	
        color: black;
		
    }</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/ram.png</normaloff>icons/ram.png</iconset>
       </attribute>
       <attribute name="title">
        <string>OPC Memories</string>
       </attribute>
       <widget class="QLabel" name="lCounter">
        <property name="geometry">
         <rect>
          <x>275</x>
          <y>115</y>
          <width>81</width>
          <height>56</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>56</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
        QLabel {
            background-color: white;
            color: black;
            padding: 5px;
            border-radius: 5px;
        }</string>
        </property>
        <property name="text">
         <string>00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>130</y>
          <width>151</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>CONTADOR</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lFlag">
        <property name="geometry">
         <rect>
          <x>275</x>
          <y>185</y>
          <width>81</width>
          <height>56</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>56</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
        QLabel {
            background-color: white;
            color: black;
            padding: 5px;
            border-radius: 5px;
        }</string>
        </property>
        <property name="text">
         <string>True</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>200</y>
          <width>151</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>FLAG</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_toggle">
        <property name="geometry">
         <rect>
          <x>65</x>
          <y>270</y>
          <width>340</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>340</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>340</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
             QPushButton {
                 
				
				background-color: rgb(195, 255, 0);
                 color: black;
                 padding: 10px;
                 border-radius: 10px;
             }
             QPushButton:hover {
                 background-color: #a6a6a6;
             }
         </string>
        </property>
        <property name="text">
         <string>Teste Flag</string>
        </property>
        <property name="iconSize">
         <size>
          <width>33</width>
          <height>33</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="LogOpc">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>476</width>
          <height>91</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    border: 1px solid white;
    padding: 5px;
    color: white;
    background-color: #222; /* opcional, melhora contraste */
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/internet-connection.png</normaloff>icons/internet-connection.png</iconset>
       </attribute>
       <attribute name="title">
        <string>MQTT</string>
       </attribute>
       <widget class="QLabel" name="status_par">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>475</y>
          <width>466</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="HeighLine_2">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>10</y>
          <width>96</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>IP ADD:</string>
        </property>
       </widget>
       <widget class="QLabel" name="HeighLine_3">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>45</y>
          <width>46</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>LOG:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="IPserver">
        <property name="geometry">
         <rect>
          <x>555</x>
          <y>15</y>
          <width>160</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QSpinBox" name="PortS">
        <property name="geometry">
         <rect>
          <x>555</x>
          <y>50</y>
          <width>160</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="keyboardTracking">
         <bool>true</bool>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>502</number>
        </property>
       </widget>
       <widget class="QLabel" name="addbroker">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>10</y>
          <width>256</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>000.000.000.000</string>
        </property>
       </widget>
       <widget class="QLabel" name="LogMqtt">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>75</y>
          <width>466</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    border: 1px solid white;
    padding: 5px;
    color: white;
    background-color: #222; /* opcional, melhora contraste */
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="StatusMQTT">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>295</y>
          <width>400</width>
          <height>41</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>10</number>
        </property>
       </widget>
       <widget class="QLabel" name="PortB">
        <property name="geometry">
         <rect>
          <x>435</x>
          <y>10</y>
          <width>51</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>1883</string>
        </property>
       </widget>
       <widget class="QLabel" name="HeighLine_4">
        <property name="geometry">
         <rect>
          <x>385</x>
          <y>10</y>
          <width>46</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>PORT:</string>
        </property>
       </widget>
       <widget class="QLabel" name="LogMqttMsg">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>195</y>
          <width>466</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    border: 1px solid white;
    padding: 5px;
    color: white;
    background-color: #222; /* opcional, melhora contraste */
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="HeighLine_5">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>160</y>
          <width>46</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>MSG:</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
